at least one of the test fails looks real:
```
======================================================================
FAIL: test_prod_config (testapps.test_elasticsearch.tests.test_prod_es_indices.ProdIndexManagementTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/testapps/test_elasticsearch/tests/test_prod_es_indices.py", line 27, in test_prod_config
    self.assertEqual(EXPECTED_PROD_INDICES, found_prod_indices)
AssertionError: Lists differ: [{'alias': 'case_search', 'typ... != [{'alias': u'case_search', 'ty...
First differing element 6:
{'alias': 'report_cases', 'type': 'report_case', 'index': 'test_report_cases_czei39du507m9mmpqk3y01x72a3ux4p0'}
{'alias': u'ledgers', 'type': u'ledger', 'index': u'test_ledgers_2016-03-15'}
Second list contains 1 additional elements.
First extra element 10:
{'alias': u'xforms', 'type': u'xform', 'index': u'test_xforms_2016-03-02'}
```

--- a/testapps/test_elasticsearch/tests/test_prod_es_indices.py
+++ b/testapps/test_elasticsearch/tests/test_prod_es_indices.py
@@ -59,6 +59,11 @@ EXPECTED_PROD_INDICES = [
         "type": "user"
     },
     {
+        "alias": "ledgers",
+        "index": "test_ledgers_2016-03-15",
+        "type": "ledger"
+    },
+    {
         "alias": "report_cases",
         "index": "test_report_cases_czei39du507m9mmpqk3y01x72a3ux4p0",
         "type": "report_case"

--- a/testapps/test_pillowtop/tests/test_settings.py
+++ b/testapps/test_pillowtop/tests/test_settings.py
@@ -2,6 +2,7 @@ import os
 from django.conf import settings
 from django.test import TestCase, override_settings
 import json
+
 from corehq.util.test_utils import TestFileMixin
 from pillowtop.listener import AliasedElasticPillow, BasicPillow
 from pillowtop.utils import get_all_pillow_configs
@@ -9,7 +10,6 @@ from testapps.test_pillowtop.utils import real_pillow_settings
 
 
 @override_settings(DEBUG=True)
-@real_pillow_settings()
 class PillowtopSettingsTest(TestCase, TestFileMixin):
     dependent_apps = [
         'pillowtop',
@@ -20,6 +20,13 @@ class PillowtopSettingsTest(TestCase, TestFileMixin):
     file_path = ('data',)
     root = os.path.dirname(__file__)
     maxDiff = None
+
+    def setUp(self):
+        self.settings_context = real_pillow_settings()
+        self.settings_context.__enter__()
+
+    def tearDown(self):
+        self.settings_context.__exit__(None, None, None)
 
     def test_instantiate_all(self):
         all_pillow_configs = list(get_all_pillow_configs())

